ios/RNSVGNode.m

```diff
@@ -9,14 +9,21 @@
 #import "RNSVGNode.h"
 #import "RNSVGContainer.h"
 #import "RNSVGClipPath.h"
+#import "RNSVGGroup.h"
+#import "RNSVGGlyphContext.h"
 
 @implementation RNSVGNode
 {
+    RNSVGGroup *_textRoot;
+    RNSVGGlyphContext *glyphContext;
     BOOL _transparent;
     CGPathRef _cachedClipPath;
     RNSVGSvgView *_svgView;
 }
 
+CGFloat const M_SQRT1_2l = 0.707106781186547524400844362104849039;
+CGFloat const DEFAULT_FONT_SIZE = 12;
```

Comment:
It is defined here: https://www.w3.org/TR/SVG11/text.html#FontSizeProperty
And the normative document is https://www.w3.org/TR/2008/REC-CSS2-20080411/fonts.html#propdef-font-size
and has this to say about it:
Initial value: medium
An <absolute-size> keyword refers to an entry in a table of font sizes computed and kept by the user agent. Possible values are:
[ xx-small | x-small | small | medium | large | x-large | xx-large ]

On a computer screen a scaling factor of 1.2 is suggested between adjacent indexes; if the 'medium' font is 12pt, the 'large' font could be 14.4pt. Different media may need different scaling factors. Also, the user agent should take the quality and availability of fonts into account when computing the table. The table may be different from one font family to another.

So, it should be read from the font data actually.